phpunit.xml.dist

```diff
@@ -28,8 +28,12 @@
         <include>
             <directory suffix=".php">src</directory>
         </include>
+        <exclude>
+            <directory suffix=".php">src/DataFixtures</directory>
+        </exclude>
     </source>
 
     <extensions>
+        <bootstrap class="DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension" />
```

Comment:
Keep in mind. I will remove this `extensions` element again in: https://github.com/MbinOrg/mbin/actions/runs/7949978285/job/21701747844

Because we are using PHPUnit 10, which doesn't have support for this `extensions` element anymore.

Reply:
IF that is an issue... Then I think we should down-grade PHPUnit test to v9 again.

Reply:
from my understanding it's more like phpunit 10 expects the inner element of `<extensions>` tag to be of type `<bootstrap>` (as I put it there), rather than of type `<extension>` as it used to be.

I got that snippet straight from the [dama/doctrine-test-bundle docs](https://github.com/dmaicher/doctrine-test-bundle#using-the-bundle-with-phpunit), so it should be fine with phpunit 10

also removing this block will mean that the test bindle is not active, and then we'll be back to square one regarding getting functional tests to run

Reply:
Ah yes.. I see it now! great.